### PR TITLE
Citations

### PR DESCRIPTION
**Pull request by JavalVyas2000**
## Changes proposed in this PR:
- Citation.cff file added. This will now enable users to cite pareto paper from the repository. 
- Also added the software osti.gov resource to site the software package and the version that is being used. 
- Suggestion: we should add a list of previous releases, for users to cite and document it somewhere on the repository. May be @MAZamarripa can help with this suggestion. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Comment by tarnold17** [comment]
Note that citations for previous PARETO versions can be found on EDX - https://edx.netl.doe.gov/dataset/?q=PARETO&sort=score+desc&ext_bbox=&ext_prev_extent=-151.5234375%2C7.36246686553575%2C-38.67187499999999%2C61.10078883158897

**Comment by ksbeattie** [comment]
Maybe add to the readme that there is a link on the top-right of the repo page that show other ways to cite the repo.

**Comment by tarnold17** [comment]
@JavalVyas2000 I think we should recommend PARETO users to cite this paper instead of the software itself: https://rdcu.be/cYZ5S

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,30 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - name: "PARETO contributors"
+title: "Please add a title here"
```
Comment (tarnold17) [inline CITATION.cff:5]: We should probably put something here other than "Please add a title here"

**Review by ksbeattie — APPROVED** [review]
LGTM

**Review by tarnold17 — APPROVED** [review]

**Pull request merged**